dds/DCPS/AssociationData.h

```diff
@@ -28,6 +28,7 @@ struct AssociationData {
 
   AssociationData()
     : remote_id_(GUID_UNKNOWN)
+    , participant_discovered_at_({ 0, 0})
```

Comment:
I think this fails for older compilers.  It may be easier to have a static function that returns the object.

Reply:
`MonotonicTimePoint::zero_value`?